tests/async/test_launcher.py

```diff
@@ -29,7 +29,10 @@ async def test_browser_type_launch_should_reject_all_promises_when_browser_is_cl
     await page.close()
     with pytest.raises(Error) as exc:
         await never_resolves
-    assert "Target closed" in exc.value.message
+    assert (
+        "Target closed" in exc.value.message
+        or "Target page, context or browser has been closed" in exc.value.message
+    )
```

Comment:
Which should it be? Seems like we get both which I'm not sure is intended or if it's a race. @mxschmitt

Reply:
idk!